python/lsst/meas/base/sfm.py

```diff
@@ -307,6 +308,24 @@ def run(self, measCat, exposure, noiseImage=None, exposureId=None, beginOrder=No
         else:
             noiseReplacer = DummyNoiseReplacer()
 
+        self.runPlugins(noiseReplacer, measCat, exposure, beginOrder, endOrder)
+
+    def runPlugins(self, noiseReplacer, measCat, exposure, beginOrder=None, endOrder=None):
+        """!
+        Function which calls the defined measument plugins on an exposure
```

Comment:
I know the rest of the file doesn't (yet), but this new docstring should be in numpydoc format.